policy/modules/system/ipsec.fc

```diff
@@ -11,6 +11,13 @@
 /etc/ipsec\.conf		--	gen_context(system_u:object_r:ipsec_conf_file_t,s0)
 /etc/strongswan/ipsec\.secrets.*		--	gen_context(system_u:object_r:ipsec_key_file_t,s0)
 /etc/strongswan/ipsec\.conf		--	gen_context(system_u:object_r:ipsec_conf_file_t,s0)
+/etc/strongswan/swanctl/bliss/(/.*)?	gen_context(system_u:object_r:ipsec_key_file_t,s0)
```

Comment:
Would it be possible to label whole directory /etc/strongswan/swanctl as ipsec_key_file_t ? Or are there any other subdirs which requires another label?

Reply:
Labeling the whole directory with a few exception would be anothor approach. I chose this one as in swan wiki https://wiki.strongswan.org/projects/strongswan/wiki/SwanctlDirectory
they call it a configuration directory and the credentials dirs predefined. Right after installation it contains, apart from the credential directories, a configuration file and a directory.

> The swanctl configuration directory, usually /etc/swanctl, contains swanctl.conf and a predefined
> set of sub-directories that provide file based credentials, such as private keys and certificates, which are
> read by the swanctl --load-creds command.

Reply:
I see, it make sense. Different approach would label possible future sub-dirs. Let's follow as it is in commit.